<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TopicViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Прогресс оценивания модели</string>
    </property>
    <widget class="QProgressBar" name="ProgressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Интепретируемость</string>
    </property>
    <widget class="QRadioButton" name="InterpretLowRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Низкая</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">InterpretButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="InterpretMiddleRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Средняя</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+3</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">InterpretButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="InterpretHighRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Высокая</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+4</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">InterpretButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="InterpretNoneRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Отсутствует</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">InterpretButtonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Этничность</string>
    </property>
    <widget class="QRadioButton" name="EthnicLowRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Низкая</string>
     </property>
     <property name="shortcut">
      <string>Alt+2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">EthnicButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="EthnicMiddleRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Средняя</string>
     </property>
     <property name="shortcut">
      <string>Alt+3</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">EthnicButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="EthnicHighRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Высокая</string>
     </property>
     <property name="shortcut">
      <string>Alt+4</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">EthnicButtonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="EthnicNoneRButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Отсутствует</string>
     </property>
     <property name="shortcut">
      <string>Alt+1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">EthnicButtonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующая</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrevButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущая</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя темы:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TokensList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>501</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column>
     <property name="text">
      <string>Токены</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="ActionOpen"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="ActionHelp"/>
    <addaction name="ActionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ActionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="ActionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="EthnicButtonGroup"/>
  <buttongroup name="InterpretButtonGroup"/>
 </buttongroups>
</ui>
